Classes/Domain/Model/Forum/Attachment.php

```diff
@@ -218,4 +221,24 @@ public function getDownloadCount() {
 	public function increaseDownloadCount() {
 		$this->downloadCount++;
 	}
+
+    /**
+     * Gets the whole TCA config of tx_typo3forum_domain_model_forum_attachment
+     * @return array The whole TCA config of tx_typo3forum_domain_model_forum_attachment
+     */
+    public function getTCAConfig() {
+        return $GLOBALS['TCA']['tx_typo3forum_domain_model_forum_attachment'];
+    }
+
+    /**
+     * @return TypoScriptService
+     */
+    private function getTyposcriptService()
+    {
+        if (is_null($this->typoScriptService)) {
+           $this->typoScriptService = GeneralUtility::makeInstance('\\TYPO3\\CMS\\Extbase\\Service\\TypoScriptService');
+        }
```

Comment:
Yes, but not all occurrences of Attachement are build by the objectManager and injection won't work